File: csharp/Lift/LiftSystemTest.cs
using System.Collections.Generic;
using ApprovalTests;
using Xunit;

namespace Lift
{
    public class LiftSystemTest
    {
        // TODO: enable this test and finish writing it
        // [Fact]
        public void Something()
        {
            var liftA = new Lift("A", 0);
            var lifts = new LiftSystem(new List<int>(){0, 1}, new List<Lift>{liftA}, new List<Call>());
            lifts.Tick();
            Approvals.Verify(new LiftSystemPrinter().Print(lifts));
        }
    }
}

File: java/src/test/java/codingdojo/LiftSystemTest.java
package codingdojo;

import org.junit.jupiter.api.Test;

import java.util.Arrays;
import java.util.Collections;

import static org.approvaltests.Approvals.verify;

public class LiftSystemTest {

    private LiftSystemPrinter liftSystemPrinter = new LiftSystemPrinter();

    // TODO: enable this test and finish writing it
    @Test
    public void answerRequest() {
        Lift liftA = new Lift("A", 0);
        LiftSystem lifts = new LiftSystem(Arrays.asList(0, 1),
                Collections.singletonList(liftA), Collections.emptyList());
        String toVerify = "";

        toVerify += liftSystemPrinter.print(lifts);
        toVerify += "============\n";
        liftA.requestFloor(1);

        toVerify += printAndTick(lifts);
        toVerify += printAndTick(lifts);

        lifts.tick();
        verify(toVerify);
    }

    private String printAndTick(LiftSystem lifts) {
        String toVerify = liftSystemPrinter.print(lifts);
        toVerify += "============\n";
        lifts.tick();
        return toVerify;
    }
}
